src/pages/jaeyoungLee/Main/Main.scss

```
@@ -0,0 +1,412 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+nav {
+  position: fixed;
+  width: 100%;
+  top: 0px;
+  left: 0px;
+  right: 0px;
+  background-color: #fff;
+  border-bottom: 1px solid #dbdbdb;
+  .navcontents {
+      width: 935px;
+      height: 54px;
+      background-color: #fff;
+      margin: 0 auto;
+      display: flex;
+      justify-content: space-between;
+      .navleft {
+          width: 360px;
+          .navleftcontents {
+              display: flex;
+              justify-content: flex-start;
+              .logo {
+                  font-size: 23px;
+                  margin-top: 12px;
+              }
+              .logoname {
+                  border-left: 1px solid #bbbbbb;
+                  font-family: "Lobster", cursive;
+                  margin-left: 15px;
+                  margin-top: 7px;
+                  padding-left: 15px;
+              }
+          }
+      }
+      .navcenter {
```

```suggestion
      .nav_center {
```
같이 띄어쓰기를 '-'나 '_'로 표시해주면 속성읽기가 더 수월해 질 거 같습니다!

---

```suggestion
        .nav_center {
```